zmon-controller-app/src/main/resources/templates/grafana.html

```diff
@@ -24,6 +24,19 @@
             display: none;
         }
     </style>
+
+      <!-- Opentracing -->
+      <script th:inline="javascript" th:if="${eumTracingEnabled}" src="/lib/opentracing-javascript/opentracing-browser.min.js"></script>
+      <script th:inline="javascript" th:if="${eumTracingEnabled}" src="https://raw.githubusercontent.com/lightstep/lightstep-tracer-javascript/v0.20.3/dist/lightstep-tracer.min.js"></script>
```

Comment:
Let's try to remove this one and see if the utils lib can fetch it and work with it